libcore/src/IO/Trajectories.h

```diff
@@ -54,6 +57,8 @@ class Trajectories
         _optionalOutputOptions = options;
     }
 
+    virtual void SetPrecision(unsigned int precision) { _precision = precision; }
+    virtual unsigned int GetPrecision() { return _precision; }
```

Comment:
do we need the getter? If not we could just delete it

Reply:
I removed `SetPrecision()` (--> constructor) and kept `GetPrecision()` cause `_precision` is private